import/Make.defs

```diff
@@ -71,7 +71,7 @@ ifeq ($(CONFIG_BUILD_KERNEL),y)
     LDLIBS += -lxx
   endif
 ifneq ($(CONFIG_LIBM),y)
-  LIBM = ${shell "$(CC)" $(ARCHCPUFLAGS) --print-file-name=libm.a}
+  LIBM = ${wildcard ${shell $(CC) $(ARCHCPUFLAGS) --print-file-name=libm.a}}
```

Comment:
I've reexamined the case and think that this changes are not required. But instead we probably need to rework build logic not to use `LDLIBPATH` and `LDLIBS` logic for compiler libraries , but use `EXTRA_LIBS` instead.

I will try to apply the change, but most probably next week.

Reply:
@xiaoxiang781216 let's close this PR. When I was writing comment in https://github.com/apache/incubator-nuttx/pull/5864 I was not sure if changes in apps are needed or not, but now I see that they are not needed.

Reply:
Sure.